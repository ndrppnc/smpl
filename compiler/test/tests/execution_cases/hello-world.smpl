int itest = 13;
float ftest = 1.0;
char ctest = 'c';
boolean btest = true;
string stest = "I am working fine now\n";

int main(int a, int b)
{
    /* test program */
    printf("%s", stest);
}
