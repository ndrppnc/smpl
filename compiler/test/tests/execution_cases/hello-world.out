I am working fine now
